engine/source/output/anim/generate/tensor6.F

```diff
@@ -4359,7 +4360,38 @@ SUBROUTINE TENSGPS3(ELBUF_TAB,FUNC1   ,FUNC2   ,IPARG   ,GEO     ,
            ITSH=0
          ENDIF
 C----------
-         IF(ISOLNOD == 6 .OR. ISOLNOD == 8 .OR. 
+         IF (ISOLNOD == 8.AND. JHBE<9)THEN
+c    
+           DO I=LFT,LLT
+             N = I + NFT
+             IF (KCVT /= 0) THEN
+               IF(KCVT==2)THEN
+                 GAMA(1:6) = GBUF%GAMA(JJ(1:6) + I)
+               ELSE
+                 GAMA(1)=ONE
+                 GAMA(2)=ZERO
+                 GAMA(3)=ZERO
+                 GAMA(4)=ZERO
+                 GAMA(5)=ONE
+                 GAMA(6)=ZERO
+               END IF
+             END IF
+                 N1 = ONE
```

Comment:
The indentation does not seem good. Could you review it?

Reply:
done.
thanks.